File: VisitBosnia/VisitBosnia.Model/SearchObjects/TransactionSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.SearchObjects
{
    public class TransactionSearchObject
    {
        public int? AppUserId { get; set; }
        public string? Status { get; set; }
        public bool? IncludeEventOrder { get; set; }


    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/transactions/transaction_search_object.dart
class TransactionSearchObject {
  bool? includeEventOrder;
  String? status;
  int? appUserId;

  TransactionSearchObject(
      {this.includeEventOrder, this.status, this.appUserId});

  TransactionSearchObject.fromJson(Map<String, dynamic> json) {
    includeEventOrder = json['includeEventOrder'];
    status = json['status'];
    appUserId = json['appUserId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['includeEventOrder'] = this.includeEventOrder;
    data['status'] = this.status;
    data['appUserId'] = this.appUserId;
    return data;
  }
}
